.github/workflows/test.yml

```diff
@@ -0,0 +1,26 @@
+name: Test
+on: [push, pull_request]
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        node: [12, 14]
```

Comment:
the original is `stable`, we want only one here? 12? 14?

Reply:
Only 14 is fine. The node version doesn't really matter, the only concern is that `airtap` (which runs the browser tests) supports that node version.